### PR TITLE
canu 1.8 foss-2018b

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/canu/canu-1.7-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description = """Canu is a fork of the Celera Assembler designed for
+ high-noise single-molecule sequencing"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/marbl/canu/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c5be54b0ad20729093413e7e722a19637d32e966dc8ecd2b579ba3e4958d378a']
+
+dependencies = [('Boost', '1.68.0', '-Python-2.7.15')]
```
Comment (boegel) [inline easybuild/easyconfigs/c/canu/canu-1.7-foss-2018b.eb]: @fizwit Please stick to Boost 1.67.0 in the 2018b generation of easyconfigs (see failing test).

File: easybuild/easyconfigs/c/canu/canu-1.8-foss-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+description = """Canu is a fork of the Celera Assembler designed for
+ high-noise single-molecule sequencing"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/marbl/canu/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['30ecfe574166f54f79606038830f68927cf0efab33bdc3c6e43fd1448fa0b2e4']
+
+dependencies = [('Boost', '1.67.0')]
```
Comment (boegel) [inline easybuild/easyconfigs/c/canu/canu-1.8-foss-2018b.eb]: @fizwit I've had a PR open for a while for this, see #7333

I used different dependencies, not including `Boost`. Any reason you didn't include `Perl` or `gnuplot`?

Comment (fizwit) [reply easybuild/easyconfigs/c/canu/canu-1.8-foss-2018b.eb]: This is an update of something I built a few years ago.  I have never used the software so can not comment. It builds without error, but have no idea how it functions. If you push your 2018b version I will use it.

Comment (boegel) [reply easybuild/easyconfigs/c/canu/canu-1.8-foss-2018b.eb]: @fizwit I tested the installation of `canu` without having Boost available (either as a dependency, or on the system), so as far as I can tell it doesn't require `Boost`.

@verdurin Shall we go ahead and merge #7333 based on this feedback, so @fizwit can update this PR to be in sync with #7333?

Comment (boegel) [reply easybuild/easyconfigs/c/canu/canu-1.8-foss-2018b.eb]: @fizwit Ping on updating this PR to be in sync with #7333?

**Comment by fizwit** [comment]
withdrawing from 2018b.  Will use existing package canu-1.8-foss-2018b-Perl-5.28.0.eb

**Pull request closed without merging**